api/v1alpha1/discoveredcluster_types.go

```diff
@@ -28,6 +28,7 @@ type DiscoveredClusterSpec struct {
 	Name              string       `json:"name" yaml:"name"`
 	DisplayName       string       `json:"displayName" yaml:"displayName"`
 	Console           string       `json:"console,omitempty" yaml:"console,omitempty"`
+	APIURL            string       `json:"apiUrl,omitempty" yaml:"apiUrl,omitempty"`
```

Comment:
Was this intended to be a required field? If so should we remove omitempty?

Reply:
I've updated it to be required. It may still be an empty string, but it won't be omitted.